src/Services/DatabaseMigration.php

```diff
@@ -218,7 +218,7 @@ private function version20180425203400(): void
     {
         $clientTableName = $this->database->applyPrefix(ClientRepository::TABLE_NAME);
         $this->database->write(<<< EOT
-        ALTER TABLE ${clientTableName}
+        ALTER TABLE {$clientTableName}
```

Comment:
I think the DB migration change should be a different PR.  This PR should only be focussed on fixing things for SPA using auth code and pkce

Reply:
done